spec/src/test/java/impl/AnswerResourceTest.java

```diff
@@ -87,6 +123,12 @@ public void shouldMarkBothQuestionAndAnswerAsAcceptedWhenUserAcceptsAnAnswer() {
         assertThat(questionFromDb.isAnswerAccepted()).isTrue();
     }
 
+    private void assertLayoutBlock(LayoutBlock firstLayoutBlock, String wantedText) {
+        assertThat(firstLayoutBlock).isInstanceOf(SectionBlock.class);
```

Comment:
if we're more interested in asserting the text that in the types, we could do `assertThat(firstLayoutBlock).extracting("text.text").containsExactly(wantedText);`

Reply:
Snyggt!